src/components/NavigationStepper/__tests__/index.test.tsx

```diff
@@ -0,0 +1,106 @@
+import React from 'react'
+import NavigationStepper from '..'
+import shouldMatchEmotionSnapshot from '../../../helpers/shouldMatchEmotionSnapshot'
+
+describe('NavigationStepper', () => {
+  beforeAll(() => {
+    Object.defineProperty(window, 'matchMedia', {
+      value: jest.fn().mockImplementation((query: string) => ({
+        addEventListener: jest.fn(),
+        dispatchEvent: jest.fn(),
+        matches: false,
+        media: query,
+        onchange: null,
+        removeEventListener: jest.fn(),
+      })),
+      writable: true,
+    })
+  })
+
+  test('should render correctly', async () =>
+    shouldMatchEmotionSnapshot(
+      <NavigationStepper>
+        <NavigationStepper.Step>First</NavigationStepper.Step>
+        <NavigationStepper.Step>Second</NavigationStepper.Step>
+        <NavigationStepper.Step>Third</NavigationStepper.Step>
+      </NavigationStepper>,
+    ))
+
+  test('should render correctly with step prop', async () =>
+    shouldMatchEmotionSnapshot(
+      <NavigationStepper step={2}>
+        <NavigationStepper.Step>First</NavigationStepper.Step>
+        <NavigationStepper.Step>Second</NavigationStepper.Step>
+        <NavigationStepper.Step>Third</NavigationStepper.Step>
+      </NavigationStepper>,
+    ))
+
+  test('should render correctly with isLoading', async () =>
+    shouldMatchEmotionSnapshot(
+      <NavigationStepper step={2}>
+        <NavigationStepper.Step isLoading>First</NavigationStepper.Step>
+        <NavigationStepper.Step isLoading>Second</NavigationStepper.Step>
+        <NavigationStepper.Step isLoading>Third</NavigationStepper.Step>
+      </NavigationStepper>,
+    ))
+
+  test('should render correctly with condensed', async () =>
+    shouldMatchEmotionSnapshot(
+      <NavigationStepper condensed step={2}>
+        <NavigationStepper.Step isLoading>First</NavigationStepper.Step>
+        <NavigationStepper.Step>Second</NavigationStepper.Step>
+        <NavigationStepper.Step>Third</NavigationStepper.Step>
+      </NavigationStepper>,
+    ))
+
+  test('should render correctly with default condensed', async () => {
+    Object.defineProperty(window, 'matchMedia', {
```

Comment:
Already done in beforeAll ?

Reply:
But I switch from matches from false to true to emulate the default condensed at true for small screens